### DISCUSSION TITLE
Switching from Projects Classic brings unexpected inconveniences

### DISCUSSION THREAD

**Discussion by sunduchkov** [community/community#125581] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

We've been working with Project Classic for several years and love its simplicity. 

But suddenly we were forced to switch to a new project style and found the inconvenience. 

In the table view (Kanban) we cannot see who opened the task. This feature was very useful for our team. Is there any way to get this feature back?

**Comment by Viktor-Ivliev** [comment]
It's just as inconvenient with this bug: https://github.com/orgs/community/discussions/125226
And also in the fact that there is no possibility to customize the sorting variant for each column different, for example, in the classic version of the column `done` we had at the top of the last closed tasks and at the bottom of the old ones.

**Comment by evi-liu** [comment]
Hi @sunduchkov, thank you for your feedback. This is a known limitation of new projects, and we are looking at how we can bring the projects activity log to new projects. We will circle back here once we have the feature available on the roadmap.
